bindings/core/src/method_handler/utils.rs

```diff
@@ -70,7 +70,8 @@ pub(crate) fn call_utils_method_internal(method: UtilsMethod) -> Result<Response
             Response::TokenId(TokenId::from(foundry_id))
         }
         UtilsMethod::HashTransaction { transaction } => {
-            Response::Hash(prefix_hex::encode(Transaction::try_from_dto(transaction)?.hash()))
+            // TODO: Should this be the signing_hash?
```

Comment:
I don't see why bindings would need the Hash. Let's make it SigningHash for now and we can add Hash if needed. I'd rename the variant then.

Reply:
The method variant, not the response *